prescreeners/form-controls.js

```diff
@@ -63,50 +66,27 @@ const FORM_CONTROLS = {
-    'resultExplanation': document.getElementById('result-explanation'),
-    'monthly_job_income': document.getElementById('monthly_job_income'),
-    'monthly_job_income_error_elem': document.getElementById('monthly_job_income_error_elem'),
-    'monthly_non_job_income': document.getElementById('monthly_non_job_income'),
-    'monthly_non_job_income_error_elem': document.getElementById('monthly_non_job_income_error_elem'),
-    'resources': document.getElementById('resources'),
-    'dependent_care_costs': document.getElementById('dependent_care_costs'),
-    'medical_expenses_for_elderly_or_disabled': document.getElementById('medical_expenses_for_elderly_or_disabled'),
-    'rent_or_mortgage': document.getElementById('rent_or_mortgage'),
-    'homeowners_insurance_and_taxes': document.getElementById('homeowners_insurance_and_taxes'),
-    'utility_costs': document.getElementById('utility_costs'),
-    'resources_error_elem': document.getElementById('resources_error_elem'),
-    'dependent_care_costs_error_elem': document.getElementById('dependent_care_costs_error_elem'),
-    'medical_expenses_for_elderly_or_disabled_error_elem': document.getElementById('medical_expenses_for_elderly_or_disabled_error_elem'),
-    'rent_or_mortgage_error_elem': document.getElementById('rent_or_mortgage_error_elem'),
-    'homeowners_insurance_and_taxes_error_elem': document.getElementById('homeowners_insurance_and_taxes_error_elem'),
-    'utility_costs_error_elem': document.getElementById('utility_costs_error_elem'),
-    'court_ordered_child_support_payments': document.getElementById('court_ordered_child_support_payments'),
-    'court_ordered_child_support_payments_error_elem': document.getElementById('court_ordered_child_support_payments_error_elem'),
-};
-
 const FORM_SUBMIT_FUNCS = {
     'sendData': function () {
-        const jsonData = {
-            'household_size': document.getElementById('household_size').value,
-            'household_includes_elderly_or_disabled': document.querySelector('input[name="household_includes_elderly_or_disabled"]:checked').value,
-            'monthly_job_income': document.getElementById('monthly_job_income').value,
-            'monthly_non_job_income': document.getElementById('monthly_non_job_income').value,
-            'resources': document.getElementById('resources').value,
-            'dependent_care_costs': document.getElementById('dependent_care_costs').value,
-            'medical_expenses_for_elderly_or_disabled': document.getElementById('medical_expenses_for_elderly_or_disabled').value,
-            'rent_or_mortgage': document.getElementById('rent_or_mortgage').value,
-            'homeowners_insurance_and_taxes': document.getElementById('homeowners_insurance_and_taxes').value,
-            'utility_costs': document.getElementById('utility_costs').value,
-        };
+        const jsonData = {};
+        const formElements = DOM_MANIPULATORS.getElem('prescreener-form').elements;
```

Comment:
Make the JS front-end less dependent on knowing specifics about the form HTML structure.